tools/ui/src/auth/auth.ts

```diff
@@ -47,10 +47,10 @@ async function insertLoginForm() {
       provider = "https://identity.internetcomputer.org";
     }
     if (!provider) {
-      throw new Error("Please provide internet identity url in ii parameter");
+      throw new Error("Please provide a URL to your local Internet Identity service using the `ii` query parameter");
```

Comment:
If we have a doc for deploying II locally, we can also link it here.

Reply:
Seems like we don't yet have any documentation, but the AI chatbot is actually able to answer questions about it since it was mentioned in a Discord conversation at one point. 

I'll keep this in mind in case we can eventually include a link to proper documentation.